.travis.yml

```diff
@@ -1,30 +1,17 @@
 language: ruby
-os: osx
-env: OSX=10.11
-osx_image: xcode7.3
-rvm: system
+rvm: 2.0.0
+os: linux
```

Comment:
That's not how the matrix works, unfortunately; that'll imply another matrix option with those set to the default values.